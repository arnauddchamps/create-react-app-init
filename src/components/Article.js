// TODO Article qui afffiche une ligne du panier

import React from 'react';
import { ListGroupItem } from 'reactstrap';

const Article = () => <ListGroupItem>Article</ListGroupItem> 

export default Article
