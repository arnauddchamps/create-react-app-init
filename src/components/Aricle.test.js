import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Article from './Article';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Article />, div);
  ReactDOM.unmountComponentAtNode(div);
});


it('renders correctly', () => {
  const tree = renderer
    .create(<Article />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});