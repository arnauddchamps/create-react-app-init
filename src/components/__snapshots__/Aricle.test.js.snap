// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  className="list-group-item"
>
  Article
</li>
`;
